README.md

```diff
@@ -88,7 +89,7 @@ The default configuration options provided can be overriden and/or extended in o
 
 #### Repository
 
-By providing a `package-manager.json` file in the root of your repository, in the same location as your `package.json` file. Configuration options defined here will extend/override those in the default (with the exception of the `toolkitsDirectory` and `packagesDirectory` options), and are inherited by _all_ toolkits.
+By providing a `package-manager.json` file in the **`toolkits`** folder of your repository. Configuration options defined here will extend/override those in the default (with the exception of the `toolkitsDirectory` and `packagesDirectory` options), and are inherited by _all_ toolkits.
```

Comment:
Does specifying the root make it clearer?
"By providing a `package-manager.json` file at the root of the **`toolkits`** folder" of your repository.